api/specs/web-server/_wallets.py

```diff
@@ -7,6 +7,7 @@
 # pylint: disable=too-many-arguments
 
 
+from enum import Enum
```

Comment:
how did this even go through linters and mypy? I guess it is not checked?

Reply:
see `_extra_tags: list[str | Enum] = ["groups"]` below